product_docs/docs/efm/4/efm_quick_start/index.mdx

```diff
@@ -83,38 +83,38 @@ Start the configuration process on a primary or standby node. Then, copy the con
 
   The Failover Manager agent doesn't validate the addresses in the `efm.nodes` file. The agent expects that some of the addresses in the file can't be reached (for example, that another agent hasn’t been started yet).
 
-1. Configure the other nodes. Copy the `efm.properties` and `efm.nodes` files to `/etc/edb/efm-4.10` on the other nodes in your sample cluster. After copying the files, change the file ownership so the files are owned by efm:efm. The `efm.properties` file can be the same on every node, except for the following properties:
+1. Configure the other nodes. Copy the `efm.properties` and `efm.nodes` files to `/etc/edb/efm-5.0` on the other nodes in your sample cluster. After copying the files, change the file ownership so the files are owned by efm:efm. The `efm.properties` file can be the same on every node, except for the following properties:
 
    -   Modify the `bind.address` property to use the node’s local address.
 
    -   Set `is.witness` to `true` if the node is a witness node. If the node is a witness node, the properties relating to a local database installation are ignored.
 
-1. Start the Failover Manager cluster. On any node, start the Failover Manager agent. The agent is named `edb-efm-4.10`; you can use your platform-specific service command to control the service. For example, on a RHEL 7.x or Rocky Linux/AlmaLinux/RHEL 8.x host, use the command:
+1. Start the Failover Manager cluster. On any node, start the Failover Manager agent. The agent is named `edb-efm-5.0`; you can use your platform-specific service command to control the service. For example, on a RHEL 7.x or Rocky Linux/AlmaLinux/RHEL 8.x host, use the command:
 
    ```shell
-   systemctl start edb-efm-4.10
+   systemctl start edb-efm-5.0
    ```
 
 1. After the agent starts, run the following command to see the status of the single-node cluster. The addresses of the other nodes appear in the `Allowed node host` list.
 
    ```shell
-   /usr/edb/efm-4.10/bin/efm cluster-status efm
+   /usr/edb/efm-5.0/bin/efm cluster-status efm
    ```
 
 1. Start the agent on the other nodes. Run the `efm cluster-status efm` command on any node to see the cluster status.
 
-  If any agent fails to start, see the startup log for information about what went wrong:
+  If any agent fails to start, see the systemctl status for information about what went wrong. If using Failover Manager version 4.x, see the startup log instead:
 
    ```shell
-   cat /var/log/efm-4.10/startup-efm.log
+   cat /var/log/efm-5.0/startup-efm.log
```

Comment:
shouldn't this be `cat /var/log/efm-4.<x>/startup-efm.log` ?

Reply:
Yep, good catch. I haven't swept everything yet since there's a lot to do. I hopefully would have caught it. :)